index.html

```diff
@@ -12,7 +12,71 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <nav class="top-navigation">
+        <div class="content">
+            <ul class="links">
+                <li class="active">Meet Karma</li>
+                <li>How it Works</li>
+                <li>Store</li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+            <img class="logo"  src="img/karma-logo.svg" alt="Logo Image" width="20"/>
+        </div>
+    </nav>
+    <section class="section-1">
+        <div class="content">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+            <div class="buttons">
+                <button>Learn More</button>
+            </div>
+        </div>
+    </section>
+    <section class="section-2">
+        <div class="content">
+            <h3>Everyone needs a little Karma.</h3>
+            <ul class="devices">
+                <li>
+                    <img width="100" height="100" src="img/icon-devices.svg" alt="Internet for all devices image">
+                    <div class="caption">
+                        <h4>Internet for all devices</h4>
+                    </div>
+                </li>
+                <li>
+                    <img width="100" height="100" src="img/icon-coffee.svg" alt="Boost your productivity image">
+                    <div class="caption">
+                        <h5>Boost your productivity</h5>
+                    </div>
+                </li>
+                <li>
+                    <img width="100" height="100" src="img/icon-refill.svg" alt="Pay as You Go image">
+                    <div class="caption">
+                        <h6>Pay as You Go</h6>
```

Comment:
Why are these 3 headlines all using different HTML tags, when they all look the same in the original design? 
Try to keep things consistent.